public/js/utils/redux/middleware/promise.js

```diff
@@ -19,34 +20,35 @@ function promiseMiddleware({ dispatch, getState }) {
       return next(action);
     }
 
-    const promiseInst = action[PROMISE];
-    const seqId = seqIdGen().toString();
-
-    // Create a new action that doesn't have the promise field and has
+    // Mutate the action, removing the promise field and adding
```

Comment:
Don't say mutate. We aren't actually mutating the action, just changing a local variable reference. The previous comment is probably fine.